src/content/docs/curriculum-help.mdx

```diff
@@ -150,6 +225,105 @@ The `.getStyleRule()` method takes a CSS selector and returns the CSS Style Decl
 
 The `.getStyleSheet()` method returns the camper's CSS, parsed into a CSS Style Sheet object.
 
+#### `.selectorsFromSelector()`
+
+The `.selectorsFromSelector` returns all equivalent selectors from a given selector based on what's nearby
+the given element.
+
+For example if `a` is passed in a selector, it will return an array filled with several alternate selectors based
+on what's nearby. The selectors could include `a`, `label > a` , `label a` , `form label a` and `body form label a`.
+
+## JS Helper
+
+### Mockers
+
+These mock methods that can prove to be unpredictable for testing purposes.
+
+#### RandomMocker
+
+Mocks Math.random for testing purposes. Each time mock() is called the pseudo-random number generator is reset to its initial state, so that the same sequence of random numbers is generated each time. restore() restores the native Math.random function.
+
+```js
+const randomMocker = new RandomMocker();
+randomMocker.mock();
+Math.random(); // first call is always 0.2523451747838408
+Math.random(); // second call is always 0.08812504541128874
+randomMocker.mock();
+Math.random(); // generator is reset, so we get 0.2523451747838408 again
+randomMocker.restore();
+Math.random(); // back to native Math.random
+```
+
+### Static Methods
+
+There are a few methods on the `__helpers` object to retrieve data from JavaScript methods.
+
+These do NOT need to be instantiated they are static methods.
+
+#### `.isCalledWithNoArgs()`
+
+This method checks the current code string to see if a function called with any arguments.
+
+If so, true is returned. Otherwise it returns false. Additionally, if the function is commented out, false
```

Comment:
```suggestion
If so, `true` is returned. Otherwise it returns `false`. Additionally, if the function call is commented out, `false` is returned.
```